_includes/cv/map.html

```diff
@@ -1,8 +1,8 @@
-<table class="table table-sm table-borderless table-responsive">
-    {% for content in entry.contents %}
-      <tr>
-        <td class="p-1 pr-2 font-weight-bold"><b>{{ content.name }}</b></td>
-        <td class="p-1 pl-2 font-weight-light text">{{ content.value }}</td>
-      </tr>
-    {% endfor %}
-</table>
\ No newline at end of file
+<table class="table table-sm table-borderless table-responsive" style="background-color: transparent; border: none; color: var(--global-text-color);">
```

Comment:
do these styles have to be in-line or you can move them to `_sass/base.scss`?

Reply:
I think the classes like `table`, `table-sm`, etc. might be used in more than one place. I wasn't sure if this effect is something we want to apply globally. Tables could also appear in posts or other places.

Intuitively, it should be what we want since people who use the exact tables in dark mode would probably see nothing? But I can't be sure. I'll defer to your judgment on this.

Reply:
or actually we could create a `table-cv-map` class and then assign the styles there in `_sass/base.scss`?

Reply:
yeah, let's create a separate class and assign it these styles.